extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcProvider.java

```diff
@@ -59,6 +59,7 @@ public class OidcProvider implements Closeable {
     final String[] audience;
     final Map<String, String> requiredClaims;
     final Key tokenDecryptionKey;
+    final boolean verifyUserAuthSuccessWithUserInfo;
```

Comment:
I don't really use OidcProvider to optimize access to the config properties, IMHO it can be checked directly in OidcIdentityProvider